packages/ember-runtime/lib/mixins/enumerable.js

```diff
@@ -27,6 +26,12 @@ import {
 } from 'ember-metal/events';
 import compare from 'ember-runtime/compare';
 
+let _emberA;
+
+function emberA() {
+  return (_emberA || (_emberA = require('ember-runtime/system/native_array').A))();
```

Comment:
I believe that this module should also `import require from 'require';` (even if just for consistency).